client/src/actions/hanabiActions.js

```diff
@@ -0,0 +1,30 @@
+export function resetGame(){
+  return {type: "RESET_GAME"}
+}
+
+export function addPlayer(playerName){  
+  return {type: "ADD_PLAYER", playerName}
+}
+
+export function startGame(originalPlayers, originalDeck){
+  const deck = [...originalDeck];
+  let cardCount = undefined;
+  if (originalPlayers.length > 1 && originalPlayers.length < 4){
```

Comment:
we could add this method to our lib folder 
```javascript
const betweenRange = (number, lowerRange, upperRange) => {
  const min = Math.min.apply(Math, [lowerRange, upperRange]);
  const max = Math.max.apply(Math, [lowerRange, upperRange]);
  return number > min && number < max;
};
``` 

It could be useful if we are going to repeating this logic again.